Sources/Apodini/Modifier/DescriptionModifier.swift

```diff
@@ -0,0 +1,37 @@
+//
+// Created by Lorena Schlesinger on 10.01.21.
+//
+
+struct DescriptionContextKey: ContextKey {
```

Comment:
As #115 is now merged you could move to the `OptionaContextKey` protocol, instead of using an empty string as a default value 👍